MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -85,7 +85,7 @@ class RouteManeuverViewController: UIViewController {
         let stepProgress = routeProgress.currentLegProgress.currentStepProgress
         let distanceRemaining = stepProgress.distanceRemaining
         
-        distance = secondsRemaining > 5 ? distanceRemaining : nil
+        distance = distanceRemaining > 10 ? distanceRemaining : nil
```

Comment:
On a highway going 65 mph, this basically guarantees that we won’t hide the distance at all. Is that OK?